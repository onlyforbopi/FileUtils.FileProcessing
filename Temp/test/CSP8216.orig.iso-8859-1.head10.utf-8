0ÄÅÇ Á.Å.                 @DEH           ÓÔÏÉ×ÅÉÁ ÄÖ/ÄÔ/ÔÁÐ                      15022016                                                             
110000060901ÓÏÕÇÄÉÁÓ                 6ÁÈÇÍÁ          1067601001102  0000000000000001001042  19940000000   0100Ä000000000000000000000015022016         
110000088502ÁÄÑÉÁÍÏÕ                47ÁÈÇÍÁ          1055501001102  0000053000005301001252  19500000053ÉÓÏ0100Ä000000000000000000000015022016         
110000098202ÍÏÔÁÑÁ                   4ÁÈÇÍÁ          1068301001101  0000120000012001001291  19500000120   0100Ä000000000000000000000015022016         
110000368007ÂÁËÔÅÔÓÉÏÕ              49ÁÈÇÍÁ          1068101001102  0000256000025601001312  19500000256   0100Ä000000000000000000000015022016         
110000654501ÌÐÏÕÊÏÕÂÁËÁ             21ÁÈÇÍÁ          1147501001101  0000064000006401007171  19500000064   0100Ä000000000000000000000015022016         
110000671905ÊÁÐÏÄÉÓÔÑÉÏÕ            22ÁÈÇÍÁ          1068201001102  0000306000030601001292  19940000306   0100Ä000000000000000000000015022016         
110000983002ÐÁÎÙÍ                   14ÁÈÇÍÁ          1136201001101  0000114000011401006061  19500000114   0100Ä000000000000000000000015022016         
110001181601ÆÁÉÌÇ                    9ÁÈÇÍÁ          1068201001101  0000123000012301001291  19500000136   0100Ä000000000000000000000015022016         
110001189301ÆÁÉÌÇ                    9ÁÈÇÍÁ          1068201001102  0000126000012601001292  19500000126   0100Ä000000000000000000000015022016         
