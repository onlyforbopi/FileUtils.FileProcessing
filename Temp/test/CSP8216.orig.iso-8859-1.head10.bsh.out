STEP  PGM  ChkDFTAP  bash  ...
0дег а.е.                 @DEH           стоивеиа дж/дт/тап                      15022016                                                             
150
110000060901соугдиас                 6ахгма          1067601001102  0000000000000001001042  19940000000   0100д000000000000000000000015022016         
150
110000088502адяиамоу                47ахгма          1055501001102  0000053000005301001252  19500000053исо0100д000000000000000000000015022016         
150
110000098202мотаяа                   4ахгма          1068301001101  0000120000012001001291  19500000120   0100д000000000000000000000015022016         
150
110000368007бактетсиоу              49ахгма          1068101001102  0000256000025601001312  19500000256   0100д000000000000000000000015022016         
150
110000654501лпоуйоубака             21ахгма          1147501001101  0000064000006401007171  19500000064   0100д000000000000000000000015022016         
150
110000671905йаподистяиоу            22ахгма          1068201001102  0000306000030601001292  19940000306   0100д000000000000000000000015022016         
150
110000983002паным                   14ахгма          1136201001101  0000114000011401006061  19500000114   0100д000000000000000000000015022016         
150
110001181601фаилг                    9ахгма          1068201001101  0000123000012301001291  19500000136   0100д000000000000000000000015022016         
150
110001189301фаилг                    9ахгма          1068201001102  0000126000012601001292  19500000126   0100д000000000000000000000015022016         
150
